### DISCUSSION TITLE
Keep all filter options visible after applying some options

### DISCUSSION THREAD

**Discussion by mvenghaus** [Smile-SA/elasticsuite#2461] [Q&A]
Hello,

after applying a filter in a category, the options with zero results are not shown anymore in the list. But this behavior has some side effects so that i want to keep all filter options visible like there is no filter set.

The problem:
- i select the filter "product_type" and select 2 types
- after that I set the "brand" filter
-> the brand I selected doesn't have products with both types, so that one of my product_type selection is no longer visible and is lost when the user set's another filter

Does anyone have an idea how to keep all base filter options (from current category or search) visible?

Regards
Marcus

**Comment by vahonc** [comment]
Hello @mvenghaus,

At this moment, from out of the box, it's not possible to keep all filter options visible like there is no filter set. And if you don't need the rest of the features that our module provides, you should look for some other extension.

BR,
Vadym
